blocks/block-icon/index.js

```diff
@@ -5,15 +5,20 @@ import { Dashicon } from '@wordpress/components';
 import { createElement, Component } from '@wordpress/element';
 
 export default function BlockIcon( { icon } ) {
+	if ( ! icon ) {
+		return <Dashicon icon={ 'wordpress' } />;
+	}
```

Comment:
Wondering if this should be part of the `registerBlock` instead? On the fence. Do you have any thoughts on this?

Reply:
I think it's fine as it is, because [registerBlockType](https://github.com/WordPress/gutenberg/blob/25a2003c7faac2dcdb723161fefd10ef44d147b6/blocks/api/registration.js#L46) does mostly some high-level validations at the moment.

Reply:
But what if we start using it elsewhere, doing it in `registerBlockType` brings consistency.

Reply:
Both are fine, I was looking only at the currently existing code :)